test/snapshots/ui.ts.crdt

```diff
@@ -1,63 +1,71 @@
-  Scene: PUT_COMPONENT e=0x209 c=1050 t=1 data={"parent":0,"rightOf":0,"positionType":0,"alignSelf":0,"flexDirection":0,"justifyContent":0,"overflow":0,"display":0,"flexBasisUnit":0,"flexBasis":0,"flexGrow":0,"widthUnit":1,"width":700,"heightUnit":1,"height":400,"minWidthUnit":0,"minWidth":0,"minHeightUnit":0,"minHeight":0,"maxWidthUnit":0,"maxWidth":0,"maxHeightUnit":0,"maxHeight":0,"positionLeftUnit":0,"positionLeft":0,"positionTopUnit":0,"positionTop":0,"positionRightUnit":0,"positionRight":0,"positionBottomUnit":0,"positionBottom":0,"marginLeftUnit":1,"marginLeft":500,"marginTopUnit":1,"marginTop":35,"marginRightUnit":0,"marginRight":0,"marginBottomUnit":0,"marginBottom":0,"paddingLeftUnit":0,"paddingLeft":0,"paddingTopUnit":0,"paddingTop":0,"paddingRightUnit":0,"paddingRight":0,"paddingBottomUnit":0,"paddingBottom":0}
-  Scene: PUT_COMPONENT e=0x200 c=1018 t=1 data={"mesh":{"$case":"box","box":{"uvs":[]}}}
-  Scene: PUT_COMPONENT e=0x200 c=1019 t=1 data={"mesh":{"$case":"box","box":{}}}
-  OPCODES ~= 96k
-  MALLOC_COUNT = 539
-  ALIVE_OBJS_DELTA ~= 0.19k
+  Scene: PUT_COMPONENT e=0x209 c=1093 t=1 data={"placeholder":"","disabled":false}
+  Scene: PUT_COMPONENT e=0x201 c=1094 t=1 data={"acceptEmpty":false,"options":["BOEDO","CASLA"],"selectedIndex":0,"disabled":false,"color":{"r":1,"g":0,"b":0,"a":1}}
+  OPCODES ~= 117k
+  MALLOC_COUNT = 680
+  ALIVE_OBJS_DELTA ~= 0.24k
 CALL onUpdate(0.1)
   Scene: PUT_COMPONENT e=0x200 c=1 t=2 data={"position":{"x":8,"y":1,"z":8},"rotation":{"x":0,"y":0.008726535364985466,"z":0,"w":0.9999619126319885},"scale":{"x":1,"y":1,"z":1},"parent":0}
   Scene: PUT_COMPONENT e=0x204 c=1062 t=2 data={"pointerEvents":[{"eventType":1,"eventInfo":{"button":0,"showFeedback":true}}]}
-  OPCODES ~= 56k
-  MALLOC_COUNT = 211
-  ALIVE_OBJS_DELTA ~= 0.10k
+  Scene: PUT_COMPONENT e=0x20a c=1062 t=2 data={"pointerEvents":[{"eventType":1,"eventInfo":{"button":0,"showFeedback":true}},{"eventType":0,"eventInfo":{"button":0,"showFeedback":true}}]}
+  OPCODES ~= 69k
+  MALLOC_COUNT = 254
+  ALIVE_OBJS_DELTA ~= 0.12k
 CALL onUpdate(0.1)
   Scene: PUT_COMPONENT e=0x200 c=1 t=3 data={"position":{"x":8,"y":1,"z":8},"rotation":{"x":0,"y":0.017452405765652657,"z":0,"w":0.9998477101325989},"scale":{"x":1,"y":1,"z":1},"parent":0}
-  OPCODES ~= 53k
-  MALLOC_COUNT = 29
-  ALIVE_OBJS_DELTA ~= 0.01k
+  OPCODES ~= 62k
```

Comment:
Just to add context, this opcode increase should be because the adding of two components in the snapshot